<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="leftWidget" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>87</y>
      <width>152</width>
      <height>505</height>
     </rect>
    </property>
    <widget class="QListWidget" name="categoryView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>152</width>
       <height>435</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="userWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>152</width>
       <height>70</height>
      </rect>
     </property>
     <widget class="QLabel" name="userLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>12</y>
        <width>46</width>
        <height>46</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="userName">
      <property name="geometry">
       <rect>
        <x>72</x>
        <y>18</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="userLv">
      <property name="geometry">
       <rect>
        <x>72</x>
        <y>35</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="homepageWidget" native="true">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>87</y>
      <width>663</width>
      <height>479</height>
     </rect>
    </property>
    <widget class="QWidget" name="recommendWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>214</y>
       <width>529</width>
       <height>265</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="rankWidget" native="true">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>214</y>
       <width>133</width>
       <height>265</height>
      </rect>
     </property>
     <widget class="QLabel" name="rankLogo">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="rankText">
      <property name="geometry">
       <rect>
        <x>27</x>
        <y>11</y>
        <width>45</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>排行榜</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDay">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>10</y>
        <width>14</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnWeek">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>10</y>
        <width>14</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMonth">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>14</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDownTimes">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>34</y>
        <width>65</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>下载排行</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnGrade">
      <property name="geometry">
       <rect>
        <x>66</x>
        <y>34</y>
        <width>65</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>评分排行</string>
      </property>
     </widget>
     <widget class="QListWidget" name="rankView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>62</y>
        <width>133</width>
        <height>203</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="vline1">
     <property name="geometry">
      <rect>
       <x>529</x>
       <y>214</y>
       <width>1</width>
       <height>265</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="bottomWidget" native="true">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>566</y>
      <width>663</width>
      <height>26</height>
     </rect>
    </property>
    <widget class="QLabel" name="bottomImg">
     <property name="geometry">
      <rect>
       <x>438</x>
       <y>4</y>
       <width>18</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="bottomText1">
     <property name="geometry">
      <rect>
       <x>463</x>
       <y>3</y>
       <width>140</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="bottomText2">
     <property name="geometry">
      <rect>
       <x>608</x>
       <y>3</y>
       <width>50</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="upWidget" native="true">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>87</y>
      <width>663</width>
      <height>479</height>
     </rect>
    </property>
    <widget class="QListWidget" name="upListWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>79</y>
       <width>663</width>
       <height>400</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="upHeader">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>55</y>
       <width>663</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="upMSGBar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>663</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textrating_up">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>58</y>
       <width>60</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textsize_up">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>58</y>
       <width>50</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textversion_up">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>58</y>
       <width>100</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="texticon_up">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>58</y>
       <width>30</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textappname_up">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>58</y>
       <width>170</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textaction_up">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>58</y>
       <width>50</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="unWidget" native="true">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>87</y>
      <width>663</width>
      <height>479</height>
     </rect>
    </property>
    <widget class="QListWidget" name="unListWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>79</y>
       <width>663</width>
       <height>400</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="unHeader">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>55</y>
       <width>663</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="unMSGBar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>663</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textappname_un">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>58</y>
       <width>170</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textsize_un">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>58</y>
       <width>50</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textrating_un">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>58</y>
       <width>60</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textversion_un">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>58</y>
       <width>100</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="texticon_un">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>58</y>
       <width>30</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textaction_un">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>58</y>
       <width>50</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="allsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>87</y>
      <width>663</width>
      <height>479</height>
     </rect>
    </property>
    <widget class="QListWidget" name="allsListWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>79</y>
       <width>663</width>
       <height>400</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="allsHeader">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>55</y>
       <width>663</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="allsMSGBar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>663</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textsize_all">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>58</y>
       <width>50</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textrating_all">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>58</y>
       <width>60</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textappname_all">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>58</y>
       <width>170</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="texticon_all">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>58</y>
       <width>30</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textversion_all">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>58</y>
       <width>100</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textaction_all">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>58</y>
       <width>50</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="headerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>815</width>
      <height>82</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btnBack">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>31</y>
       <width>33</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNext">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>31</y>
       <width>33</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHomepage">
     <property name="geometry">
      <rect>
       <x>151</x>
       <y>0</y>
       <width>103</width>
       <height>83</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUp">
     <property name="geometry">
      <rect>
       <x>253</x>
       <y>0</y>
       <width>103</width>
       <height>83</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUn">
     <property name="geometry">
      <rect>
       <x>355</x>
       <y>0</y>
       <width>103</width>
       <height>83</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="logoImg">
     <property name="geometry">
      <rect>
       <x>659</x>
       <y>18</y>
       <width>127</width>
       <height>44</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTask">
     <property name="geometry">
      <rect>
       <x>457</x>
       <y>0</y>
       <width>103</width>
       <height>83</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMin">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>5</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConf">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>5</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="searchWidget" native="true">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>87</y>
      <width>663</width>
      <height>479</height>
     </rect>
    </property>
    <widget class="QListWidget" name="searchListWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>79</y>
       <width>663</width>
       <height>400</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="searchHeader">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>55</y>
       <width>663</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="searchMSGBar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>663</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="taskWidget" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>87</y>
      <width>815</width>
      <height>505</height>
     </rect>
    </property>
    <widget class="QLabel" name="taskHeader">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>815</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="taskMSGBar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>815</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QListWidget" name="taskListWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>54</y>
       <width>815</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="texticon">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>33</y>
       <width>30</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textappname">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>33</y>
       <width>170</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textsize">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>33</y>
       <width>50</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textprocess">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>33</y>
       <width>100</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="textstatus">
     <property name="geometry">
      <rect>
       <x>585</x>
       <y>33</y>
       <width>100</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="searchBG" native="true">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>87</y>
      <width>663</width>
      <height>33</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="leSearch">
     <property name="geometry">
      <rect>
       <x>495</x>
       <y>7</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lebg">
     <property name="geometry">
      <rect>
       <x>478</x>
       <y>7</y>
       <width>157</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="shadowleft">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>5</width>
      <height>587</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="shadowright">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>5</y>
      <width>5</width>
      <height>587</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="shadowup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>825</width>
      <height>5</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="shadowbottom">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>592</y>
      <width>825</width>
      <height>6</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
